packages/eslint-plugin-pf-codemods/src/rules/helpers/JSXAttributes.ts

```diff
@@ -15,3 +25,74 @@ export function getExpression(node?: JSXAttribute["value"]) {
     return node.expression;
   }
 }
+
+function getMemberExpression(node: MemberExpression) {
+  if (!node) {
+    return;
+  }
+  const { object, property } = node;
+
+  return { object, property };
+}
+
+export function getVariableDeclaration(
+  name: string,
+  scope: Scope.Scope | null
+) {
+  while (scope !== null) {
+    const variable = scope.variables.find((v) => v.name === name);
+
+    if (variable) {
+      return variable;
+    }
+
+    scope = scope.upper;
+  }
+  return undefined;
+}
+
+export function getVariableValue(name: string, scope: Scope.Scope | null) {
+  const variableDeclaration = getVariableDeclaration(name, scope);
+
+  if (!variableDeclaration) {
+    return;
+  }
+
+  const variableInit = variableDeclaration.defs.length
+    ? variableDeclaration.defs[0].node.init
```

Comment:
We can open a followup to address this if you'd like.

Reply:
Sure. It probably won't cause any problems for now, since I cannot imagine someone would be passing a function name (or anything other than a variable name) as an attribute value.

Opened a followup issue: #630